### DISCUSSION TITLE
Is list of of what is considered as "suffix" or "prefix" for the Lapwing steno theory in the book exhaustive?

### DISCUSSION THREAD

**Discussion by evgenyfadeev** [aerickt/lapwing-for-beginners#29] [Q&A]
If not - would it be worth to enumerate them all along with the accepted outlines?

**Comment by aerickt** [comment]
It's not exhaustive, but the ones listed _should_ be pretty much all you need.
There are some niche affixes in the dictionary carried over from main.json when usage was less strict. They're not very useful, and when it comes to words with uncommon suffixes, you can alternatively use syllabic splitting exclusively.
